build.gradle

```diff
@@ -42,8 +42,8 @@ repositories {
 }
 
 dependencies {
-    compile group: 'org.jetbrains.kotlin', name: 'kotlin-stdlib-jre8', version: '$kotlin_version'
-    compile group: 'org.jetbrains.ktor', name: 'ktor-core', version: '0.4.0'
+    compile group: 'org.jetbrains.kotlin', name: 'kotlin-stdlib-jre8', version: "1.2.21"
```

Comment:
Why not `'$kotlin-version'`? Also consider using [1.2.31](https://github.com/JetBrains/kotlin/releases/tag/v1.2.31) unless there is a specific reason not to?

Please note that there is also two dependency blocks (this one on line 45 and one starting on line 29); Making kotlin declared as a dependency twice, once as `compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"` and again as `compile group: 'org.jetbrains.kotlin', name: 'kotlin-stdlib-jre8', version: "1.2.21"`. This could eventually be problematic if the version variable is changed in the future while `kotlin-stdlib-jre8` remains at 1.2.21.

[For later versions (1.2.x)](https://kotlinlang.org/docs/reference/using-gradle.html#configuring-dependencies), the preferred dependency is _org.jetbrains.kotlin:kotlin-stdlib-**jdk**8_.